main/main.cpp

```diff
@@ -4158,6 +4158,8 @@ int Main::start() {
 
 	OS::get_singleton()->set_main_loop(main_loop);
 
+	GDExtensionManager::get_singleton()->startup();
```

Comment:
I think I would move this even later. Maybe even after starting the movie writer?